### PR TITLE
move getHostTime in getHostInfo

### PR DESCRIPTION
**Pull request by foxriver76**

### CODE REVIEW

File: packages/common/lib/common/tools.js [hunk 1]
```diff
@@ -1870,7 +1872,9 @@ async function getHostInfo(objects, callback) {
         Model: cpus && Array.isArray(cpus) ? cpus[0].model : null,
         RAM: os.totalmem(),
         'System uptime': Math.round(os.uptime()),
-        'Node.js': process.version
+        'Node.js': process.version,
+        time: dateObj.getTime(), // give infos to compare the local times
+        offset: dateObj.getTimezoneOffset()
```
Comment (Apollon77) [inline packages/common/lib/common/tools.js:1877]: better timeOffset that one can see that it belongs together?

File: README.md [hunk 2]
```diff
@@ -917,7 +917,7 @@ The following features can be checked using this method:
 * **CONTROLLER_NPM_AUTO_REBUILD**: Automatic rebuild when node version mismatch is detected (since js-controller 3.0)
 * **CONTROLLER_READWRITE_BASE_SETTINGS**: Allow read and write of js-controller base settings file (iobroker.json) via host messages (since js-controller 3.0)
 * **CONTROLLER_MULTI_REPO**: indicates that controller supports multiple repository sources, which will be combined in one (since js-controller 4.0)
-* **CONTROLLER_LICENSE_MANAGER**: js-controller can read licenses from iobroker.net (since js-controller 4.0)* **CONTROLLER_GET_TIME**: indicates that controller can send local host time and offset as response to `getHostTime` message (since js-controller 4.0)
+* **CONTROLLER_LICENSE_MANAGER**: js-controller can read licenses from iobroker.net (since js-controller 4.0)* **CONTROLLER_GET_TIME**: indicates that controller can send local host time and offset included in a response to `getHostInfo` message (since js-controller 4.0)
```
[the remark this answers is not in the document]
Comment (GermanBluefox) [reply README.md:920]: We don't need the flag

**Comment by foxriver76** [comment]
do we need the time determination in `getHostInfoShort` too?

**Comment by Apollon77** [comment]
@GermanBluefox ??

**Comment by GermanBluefox** [comment]
> do we need the time determination in `getHostInfoShort` too?

In getCompactHosts it will not harm as this information is very short.

**Comment by AlCalzone** [comment]
Just leaving a comment here so we don't forget - changing these data structures will require an update of the type definitions and maybe the socket library to have them picked up.

**Comment by Apollon77** [comment]
Added issue

**Pull request merged**